<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PredatorPreyGame</class>
 <widget class="QMainWindow" name="PredatorPreyGame">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>706</width>
    <height>500</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>PredatorPreyGame</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/PredatorPreyGame/resources/images/background/spashScreenBackground.jpg</normaloff>:/PredatorPreyGame/resources/images/background/spashScreenBackground.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWidget" name="field" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>480</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="fieldLaout"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="menuLaout">
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>70</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Menu" name="menuWidget" native="true"/>
      </item>
      <item>
       <widget class="ControlBtns" name="controlWidget" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Menu</class>
   <extends>QWidget</extends>
   <header>../widgets/menu/Menu.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ControlBtns</class>
   <extends>QWidget</extends>
   <header>../widgets/control/ControlBtns.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
